Week1/homework/js-exercises/LogNumber.js

```diff
@@ -0,0 +1,7 @@
+"use strict";
+let numberX;
+console.log("The value of numberX is undefined");
+console.log(numberX);
+numberX=66;
```

Comment:
Well done, seems to be working just fine! Small note: it's often better for readability to use space in between your assignments so: `let number = 1` instead of `let number=1`. Try to get used to writing it as such, or use a plugin like `Prettier" to clean up your syntaxing! You can ask your teacher about it!